maas/plugins/galera_check.py

```diff
@@ -74,8 +74,8 @@ def print_metrics(replica_status):
            replica_status['wsrep_commit_window'], 'sequence_delta')
     metric('wsrep_cluster_size', 'int64',
            replica_status['wsrep_cluster_size'], 'nodes')
-    metric('queries_per_second', 'int64',
-           replica_status['Queries'], 'qps')
+    metric('queries', 'int64',
```

Comment:
@sigmavirus24 @claco 
That's a good point. I honestly didn't consider anyone was using these metrics outside of what we have in place in the alarm's criteria. But even if they are using "queries_per_second" and I leave it in there for backwards compatibility, it still is kind of a misleading name( the metric reports number of queries total, not queries_per_second). I'm afraid that the mis-interpretation of the value might cause issues for anyone trying to utilize the metric. The sooner we make it unavailable, the better IMO.
What do you guys think about keeping the name change(and leaving queries_per_second out), then adding a release note documenting the change?